fix listener docs - admin api only on main process (#17590)
element-hq/synapse: docs/usage/configuration/config_documentation.md

Valid resource names are:

* `client`: the client-server API (/_matrix/client). Also implies `media` and `static`.

>   On the main process is also the synapse admin API (/_synapse/admin) loaded.

Reviewer: ```suggestion
  If configuring the main process, the Synapse Admin API (/_synapse/admin) is also implied.
```
